app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -147,7 +147,10 @@ def create
 
     # Create meeting in database
     logger.debug("Creating meeting #{params[:meetingID]} in database.")
-    meeting = Meeting.find_or_create_with_server(params[:meetingID], server)
+
+    moderator_pwd = params[:moderatorPW] || SecureRandom.alphanumeric(8)
```

Comment:
We need to handle the case of a blank value (empty string) here. Rails provides a helper for this:
```suggestion
    moderator_pwd = params[:moderatorPW].presence || SecureRandom.alphanumeric(8)
```